app/src/main/java/com/example/jellyfishandroidexample/RoomViewModel.kt

```diff
@@ -60,10 +60,9 @@ class RoomViewModel(application: Application) :
         client.join()
     }
 
-    override fun onAuthError() {
-    }
+    override fun onAuthError() {}
 
-    override fun onJoinSuccess(peerID: String, peersInRoom: List<Peer>) {
+    override fun onJoined(peerID: String, peersInRoom: List<Endpoint>) {
```

Comment:
change Endpoint to Peer also in arguments here

Reply:
and also in other functions here, in this file there shouldn't be endpoints at all other tahn ctx.endpoints